docs/installing-solidity.rst

```diff
@@ -29,105 +29,78 @@ To install it, simply use
 Details about the usage of the Node.js package can be found in the
 `solc-js repository <https://github.com/ethereum/solc-js>`_.
 
-Binary Packages
+Binary Releases
 ===============
 
-Binary packages of Solidity together with its IDE Mix are available through
-the `C++ bundle <https://github.com/ethereum/webthree-umbrella/releases>`_ of
-Ethereum.
+The `solidity-v0.3.6 <https://github.com/ethereum/solidity/releases/tag/v0.3.6>`_
+release (10th August 2016) was the first "standalone" version of Solidity.
+All runtime dependencies onto cpp-ethereum have been removed.  This was a pretty
+major refactoring.  We also transitioned from Jenkins to TravisCI and Appveyor.
 
-Building from Source
-====================
+At the time of writing, our Homebrew and ZIP workflows have not yet been
+restored.  They will be back for the pending solidity-v0.4.0 release.
 
-Building Solidity is quite similar on MacOS X, Ubuntu and probably other Unices.
-This guide starts explaining how to install the dependencies for each platform
-and then shows how to build Solidity itself.
+Ubuntu PPAs **are** working, though there are still some clashes with the
+cpp-ethereum PPAs which won't be resolved until the next cpp-ethereum release.
+This is the sequence of commands which should be successful.  We will get the
+PPAs into a state where they can cleanly coexist as soon as we can. ::
 
-MacOS X
--------
+    sudo add-apt-repository -y ppa:ethereum/ethereum
+    sudo add-apt-repository -y ppa:ethereum/ethereum-dev
+    apt-get remove libethereum solc cpp-ethereum
+    apt-get install solc
 
 
-Requirements:
+Building from Source (macOS, Debian or Ubuntu)
+==============================================
 
-- OS X Yosemite (10.10.5)
-- Homebrew
-- Xcode
+As of August 2016 we now support an **install_deps.sh** script which encodes all of the
```

Comment:
Yes, until we fix the binaries, this detailed context is important.
We're being asked repeatedly.
We can strip it back down again when we have the binaries back.